src/seals/testing/envs.py

```diff
@@ -164,15 +169,18 @@ def test_seed(env: gym.Env, env_name: str, deterministic_envs: Iterable[str]) ->
     # eventually get a different result. For deterministic environments, all
     # seeds should produce the same starting state.
     def different_seeds_same_rollout(seed1, seed2):
-        new_actions = [env.action_space.sample() for _ in range(10)]
+        new_actions = [env.action_space.sample() for _ in range(100)]
```

Comment:
Maybe allow the number of actions and the number of seeds to be arguments?

Reply:
Good idea, will do.

Reply:
Good idea, will do